README.md

```diff
@@ -1 +1,26 @@
-# to-do-list
\ No newline at end of file
+# to-do-list
+
+This project is a to-do-list that you can add or remove on the list
+and also reorder them.
+
+
+![screenshot](./Screenshot.png)
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+
+
+## Authors
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/SimonGrchevski)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/simon-grchevski-682935209/)
+
+
+
```

Comment:
- Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.